internal/git/receive.go

```
@@ -211,17 +219,17 @@ func ReceivePack(tempDir string, body []byte, owner string, repo string, cloneUR
 	cmd.Stdin = bytes.NewReader(packfile)
 
 	output, err := cmd.CombinedOutput()
-	fmt.Printf("DEBUG: git index-pack output: %s\n", string(output))
+	debugf("DEBUG: git index-pack output: %s\n", string(output))
 
 	if err != nil {
 		// Si index-pack falla, intentar unpack-objects
-		fmt.Printf("DEBUG: index-pack failed, trying unpack-objects\n")
+		debugf("DEBUG: index-pack failed, trying unpack-objects\n")
 		cmd = exec.Command("git", "unpack-objects", "-r")
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)